packages/build-ui/src/pages/agits/_id/management/collections/components/new_collection_modal.rs

```diff
@@ -110,14 +110,14 @@ pub fn NewCollectionModal(
                                             td { class: "px-4 py-3",
                                                 Checkbox {
                                                     id: format!("checkbox_{}", artwork.id),
-                                                    checked: selected_artworks.read().contains(&artwork.id),
+                                                    checked: selected_artworks.read().contains(&(artwork.id as usize)),
                                                     onchange: move |checked| {
                                                         selected_artworks
                                                             .with_mut(|vec| {
                                                                 if checked {
-                                                                    vec.push(artwork.id);
+                                                                    vec.push(artwork.id as usize);
                                                                 } else {
-                                                                    vec.retain(|&x| x != artwork.id);
+                                                                    vec.retain(|&x| x != (artwork.id as usize));
```

Comment:
how do you gurantee x is id of artwork?
I think it can be guranteed only if it's the first page.
- From the second page, it may be completely different.

Let me know how to do work on it. @sublime247